go/extractor/util/extractvendordirs.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 func IsVendorDirExtractionEnabled() bool {
-	return os.Getenv("CODEQL_EXTRACTOR_GO_EXTRACT_VENDOR_DIRS") == "true"
+	return os.Getenv("CODEQL_EXTRACTOR_GO_EXTRACT_VENDOR_DIRS") == "true" ||
+		os.Getenv("CODEQL_EXTRACTOR_GO_OPTION_EXTRACT_VENDOR_DIRS") == "true"
```

Comment:
Bit unfortunate that I didn't learn about the extractor options mechanism until after I added this and now we have two variable names for this. 

This is fine as is, but if we felt strongly about not having two variables for the same thing, perhaps we could warn if the "old" one is set to something and that `CODEQL_EXTRACTOR_GO_OPTION_EXTRACT_VENDOR_DIRS` should be used instead? I doubt this has become a popular feature, but could come to bite someone if they set one to `true` and then the other one to `false` in a workflow or script.

Reply:
OK, done